nixos/modules/installer/tools/nixos-install.sh

```diff
@@ -83,8 +83,11 @@ if [[ ! -e $NIXOS_CONFIG && -z $system ]]; then
 fi
 
 # A place to drop temporary stuff.
+tmpdir="$(mktemp -d -p $mountPoint)"
 trap "rm -rf $tmpdir" EXIT
-tmpdir="$(mktemp -d)"
+
+# store temporary files on target filesystem by default
+export TMPDIR=${TMPDIR:-$tmpdir}
```

Comment:
TMPDIR is unused?

Reply:
`TMPDIR`  is used by `nix build`.

Reply:
ok, it's a canonical variable :) Looks good then